src/cmap/connection.ts

```diff
@@ -695,7 +695,11 @@ export class CryptoConnection extends Connection {
   ): Promise<Document> {
     const { autoEncrypter } = this;
     if (!autoEncrypter) {
-      throw new MongoMissingDependencyError('No AutoEncrypter available for encryption');
+      // TODO(NODE-6065): throw a MongoMissingDependencyError in Node V7
```

Comment:
```suggestion
      // TODO(NODE-6065): throw a MongoRuntimeError in Node V7
```

according to the ticket